### PR TITLE
Update walkthrough

### PR DESCRIPTION
**Pull request by kntsoriano**
Periodic PR to update the walkthrough. This should be automated soon :)

### CODE REVIEW

File: walkthrough/index.html [hunk 1]
```diff
@@ -136,7 +136,7 @@
 
 ---
 
-## Do you know your process architecture (32-bit vs 64-bit vs ARM)?
+## Do you know your process architecture (x86 (32-bit) vs x86_64 vs ARM)?
```
Comment (SleeplessByte) [inline walkthrough/index.html]:
```suggestion
## Do you know your process architecture (x86 (32-bit) vs x86_64 (64-bit) vs ARM)?
```

File: walkthrough/index.html [hunk 2]
```diff
@@ -187,8 +187,7 @@
 
 ## Download the archive
 
-If you know your process architecture (32-bit vs 64-bit vs
-ARM), download the appropriate archive from the
+If you know your process architecture (x86 (32-bit) vs x86_64 vs ARM), download the appropriate archive from the
```
Comment (SleeplessByte) [inline walkthrough/index.html]:
```suggestion
If you know your process architecture (x86 (32-bit) vs x86_64 (64-bit) vs ARM), download the appropriate archive from the
```

**Comment by kntsoriano** [comment]
Thanks for the comments :) Will add those.

**Comment by SleeplessByte** [comment]
```html
&lt;a href="https://github.com/exercism/cli/releases/latest" target="_blank"&gt;releases page&lt;/a&gt;.
```

The escaped `<` and `>` are weirding me out XD

**Pull request merged**